platform-sdk/swirlds-jasperdb/src/test/java/com/swirlds/merkledb/files/DataFilePbjLowLevelTest.java

```diff
@@ -0,0 +1,399 @@
+    // Helper Methods
+
+    /**
+     * For tests, we want to have all different dta sizes, so we use this function to choose how
+     * many times to repeat the data value long
+     */
+    private int getRepeatCountForKey(long key) {
+        return (int) (key % 20L);
+    }
+
+    /** Create an example variable sized data item with lengths of data from 0 to 20. */
+    private long[] getVariableSizeDataForI(int i) {
+        int repeatCount = getRepeatCountForKey(i);
+        long[] dataValue = new long[1 + repeatCount];
+        dataValue[0] = i;
+        for (int j = 1; j < dataValue.length; j++) {
+            dataValue[j] = i + 10_000;
+        }
+        return dataValue;
+    }
+
+    /** Check a fixed or variable size data items data */
+    private void checkItem(FilesTestType testType, int i, long[] dataItem) {
+        switch (testType) {
+            default:
+            case fixed:
+                assertEquals(i, dataItem[0], "unexpected data item[0]");
+                assertEquals(i + 10_000, dataItem[1], "unexpected data item[1]");
+                break;
+            case variable:
+                assertEquals(
+                        Arrays.toString(getVariableSizeDataForI(i)),
+                        Arrays.toString(dataItem),
+                        "unexpected variable data");
+                break;
+        }
+    }
+
+    // =================================================================================================================
+    // Tests
+
+    @Order(2)
+    @ParameterizedTest
+    @EnumSource(FilesTestType.class)
+    void createFile(FilesTestType testType) throws IOException {
+        // open file and write data
+        DataFileWriter<long[]> writer = new DataFileWriterPbj<>(
+                "test_" + testType.name(), tempFileDir, DATA_FILE_INDEX, testType.dataItemSerializer, TEST_START);
+        LongArrayList listOfDataItemLocations = new LongArrayList(1000);
+        for (int i = 0; i < 1000; i++) {
+            long[] dataValue;
+            switch (testType) {
+                default:
+                case fixed:
+                    dataValue = new long[] {i, i + 10_000};
+                    break;
+                case variable:
+                    dataValue = getVariableSizeDataForI(i);
+                    break;
+            }
+
+            listOfDataItemLocations.add(writer.storeDataItem(dataValue));
+        }
+        writer.finishWriting();
+        final var dataFileMetadata = writer.getMetadata();
+        // tests
+        assertTrue(Files.exists(writer.getPath()), "expected file does not exist");
+        assertEquals(
+                writer.getPath(),
+                createDataFilePath("test_" + testType.name(), tempFileDir, DATA_FILE_INDEX, TEST_START, FILE_EXTENSION),
+                "unexpected path for writer");
+//        long expectedFileSizeEstimate = testType.getDataFileLowLevelTestFileSize();
```

Comment:
Please, either remove it or uncomment it.

Reply:
Removed